README.md

```diff
@@ -0,0 +1,185 @@
+# ecs_prometheus_exporter
+
+🚧 🚧 🚧 This repo is still work in progress and is subject to change.
+
+This repo contains a Prometheus exporter for
+Amazon Elastic Container Service (ECS) that publishes
+ECS task infra metrics in Prometheus format.
+
+You need to run it as a sidecar on your ECS tasks.
+
+By default, it publishes Prometheus metrics on ":9779/metrics". The exporter in this repo can be a useful complementary sidecar for the scenario described in [this blog post](https://aws.amazon.com/blogs/opensource/metrics-collection-from-amazon-ecs-using-amazon-managed-service-for-prometheus/). Adding this sidecar to the ECS task definition would export task-level metrics in addition to the custom metrics described in the blog.  
+
+The sidecar process is also supported on [AWS App Runner](https://aws.amazon.com/apprunner/)
+and can be used to publish infra metrics in Prometheus format
+from App Runner services.
+
+## Labels
+
+* **cluster**: The ECS cluster the task is running.
```

Comment:
It sounds like those labels should be exposed by the service discovery as meta labels, then relabeled with relabel_config.

This would also ensure that those target labels are consistent when users also chose to expose their metrics from their application, and help them reduce the cardinality if they do not need all those labels.

Reply:
We will still produce container and network_interface here because discovery can only provide the task labels.